nekoyume/Assets/_Scripts/L10n/Editor/L10nManagerEditor.cs

```diff
@@ -145,6 +151,151 @@ public static void GenerateUnicodeHexRangeFiles()
                 }
             }
         }
+        
+        [MenuItem("Tools/L10n/Generate Font Asset Files at Once")]
+        public static void GenerateFontAssetFiles()
+        {
+            GenerateUnicodeHexRangeFiles();
+
+            EditorCoroutineUtility.StartCoroutine(CoGenerateFontAssetFile(), EditorCoroutineObject);
+        }
+
+        private static IEnumerator CoGenerateFontAssetFile()
+        {
+            var charactersPath = Path.Combine(Application.dataPath, "Font/CharacterFiles");
+            var window = EditorWindow.GetWindow<TMPro_FontAssetCreatorWindow>();
+            
+            var languageTypes = Enum.GetValues(typeof(LanguageType)).OfType<LanguageType>();
+            foreach (var languageType in languageTypes)
+            {
+                Debug.Log($"-------------Generate Start : {languageType}-------------");
+                var settings = DefaultSettings[(int) languageType];
+                var characterPath = Path.Combine(charactersPath,
+                    $"{languageType.ToString()}-unicode-hex-range-{1:00}.txt");
+                var unicodeHexes = File.ReadAllLines(characterPath);
+                settings.characterSequence = unicodeHexes[0];
+                var fontAssetFullPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(settings.referencedFontAssetGUID))
+                    .Replace("\\", "/");
+
+                var generator = new FontAssetGenerator(window);
+                generator.GenerateAtlas(settings);
+                yield return new EditorWaitForSeconds(5.0f);
```

Comment:
What does this five-second wait do?

Reply:
If these 5 seconds are time to wait for other tasks, is this a verified number?

Reply:
To generate altas, it usually takes less than a second. 
But, If it takes a lot of time like Japanese or ChineseSimplified, it takes 1.0 to 2.5 seconds.
I calculated enough time considering the computer environment.

![image](https://user-images.githubusercontent.com/63496908/159223390-8fc8efe1-521b-4ff3-a81d-e25c00d56eb8.png)

Reply:
As you mentioned, it seems that the time may vary depending on the computer environment. Did you end up waiting for a certain amount of time because there is no other way to wait for other work?

I think it would be best to wait for the task, but if there is no way, it would be fine as it is.
Depending on the computer environment, the corresponding time may decrease or increase, so why don't you provide the const value so that you can customize the waiting time?

Reply:
I deleted the existing async/await code.

Reply:
and made wait time const too.

Reply:
LGTM 👍